js/gdm/authPrompt.js

```diff
@@ -111,6 +138,28 @@ var AuthPrompt = GObject.registerClass({
         this._message.clutter_text.line_wrap = true;
         this._message.clutter_text.ellipsize = Pango.EllipsizeMode.NONE;
         this.add_child(this._message);
+
+        let passwordResetLabel = new St.Label({
```

Comment:
```suggestion
        const passwordResetLabel = new St.Label({
```

There are lots of `let` variables that should be `const` in this file, please update them.

Reply:
Done, changed all occurrences I found.